zircon.core/src/jvmTest/kotlin/org/hexworks/zircon/internal/color/DefaultTileColorTest.kt

```diff
@@ -23,6 +24,73 @@ class DefaultTileColorTest {
         assertThat(result.alpha).isEqualTo(TileColor.defaultAlpha())
     }
 
+    @Test
+    fun testTinting() {
+        try {
```

Comment:
You can use `assertFailsWith` here instead of the `try`/`catch`:

```

@Test
fun `calling hereIsAnException will return an exception no matter what`() {
  assertFailsWith<IllegalArgumentException> { hereIsAnException() }
}
```